src/helpers/userHelper.js

```diff
@@ -1754,6 +1755,99 @@ const userHelper = function () {
       console.error('Error deleting expired time off requests:', error);
     }
   };
+  const updateProfilesPic = async () => {
+    try {
+      console.log('update picture')
+      const browser = await chromium.launch();
+      const page = await browser.newPage();
+      await page.goto('https://www.onecommunityglobal.org/team/', { waitUntil: 'networkidle' }); // Wait for network to be idle
+      
+      await page.evaluate(() => window.scrollBy(0, window.innerHeight));
+      await page.waitForTimeout(1000); 
+
+
+      const response = await page.content();
+      await browser.close();
+
+      // Now you can parse the response with Cheerio
+      const cheerio = require('cheerio');
+      const $ = cheerio.load(response);
+      const imgElements = $('img');
+      const userProfiles = await userProfile.find( {
+        // comment out the line below for testing
+        // firstName: 'yourname',
+        isActive: true
+      });
+
+      for (let profile of userProfiles) {
+        const { firstName, lastName, profilePic, isActive, email } = profile;
```

Comment:
Consider using the isActive property, as it's not being used anywhere to prevent assigning a picture to an existing user that is not active. Eg:

`if (!isActive) continue`